.github/workflows/continuous-integration-ci.yml

```diff
@@ -38,10 +38,10 @@ jobs:
         run: |
           make format
           if ! git diff --quiet; then
-            echo 'Please run `make format` on your change to apply the following formatting fixes:' >> $GITHUB_STEP_SUMMARY
-            echo '```diff' >> $GITHUB_STEP_SUMMARY
-            git diff >> $GITHUB_STEP_SUMMARY
-            echo '```' >> $GITHUB_STEP_SUMMARY
+            echo 'Please run `make format` on your change to apply the following formatting fixes:' | tee -a $GITHUB_STEP_SUMMARY
+            echo '```diff' | tee -a $GITHUB_STEP_SUMMARY
```

Comment:
This doesn't need to be `tee`'d. Ditto for line 44.

Reply:
Ack, thanks.  Updated.

Reply:
Grumble if GH doesn't have a place to do this but in a way that PR can see it.  D'oh.